ckan/patches/ckan/datasets_per_page.patch

```diff
@@ -0,0 +1,27 @@
+diff --git a/ckan/controllers/package.py b/ckan/controllers/package.py
+index fb28d93aa..78455f2eb 100644
+--- a/ckan/controllers/package.py
++++ b/ckan/controllers/package.py
+@@ -148,11 +148,11 @@ class PackageController(base.BaseController):
+         c.query_error = False
+         page = h.get_page_number(request.params)
+
+-        limit = int(config.get('ckan.datasets_per_page', 20))
++        limit = int(request.params.get('limit', config.get('ckan.datasets_per_page', 20)))
+
+         # most search operations should reset the page counter:
+         params_nopage = [(k, v) for k, v in request.params.items()
+-                         if k != 'page']
++                         if k in ['page', 'limit']]
```

Comment:
This is not needed. We want to keep the `limit` param in the pager URL (and besides it should be `not in`)